dtmi/nxp/evse-1.json

```diff
@@ -0,0 +1,244 @@
+[
```

Comment:
We require to describe a single interface per file, hence we do not support an Array.

Reply:
I Have removed the [ and ] and updated .json.  Interesting that the original model file past the validation checks.

Reply:
DTDL allows an Array as a top level element, however the device model repository has stricter rules, such as one interface per file as described [here](https://github.com/Azure/iot-plugandplay-models/blob/main/pr-reqs.md#model-file-content-structure).

If you are using the `dmr-client validate` command, make sure to use the `--strict` option to verify this case.